config.md

```diff
@@ -2,6 +2,7 @@
 
 The container's top-level directory MUST contain a configuration file called `config.json`.
 The canonical schema is defined in this document, but there is a JSON Schema in [`schema/config-schema.json`](schema/config-schema.json) and Go bindings in [`specs-go/config.go`](specs-go/config.go).
+For properties that are only defined for some [protocols](spec.md#protocols), the Go property has a `platform` tag listing those protocols (e.g. `platform:"linux,solaris"`).
```

Comment:
> I'd call them platforms not protocols, because that is what they actually are and why the tag is named that way.

That's what they are now, but `protocols` [leaves the door open for future entries that are not platforms][1].  Are we comfortable closing that door (I'd rather leave it open, but I'm not a maintainer)?

If maintainers are comfortable closing the door, how thoroughly should I replace ‘protocol’ with ‘platform’?  Do you want me to change these too:

    $ git grep -i protocol origin/master
    origin/master:spec.md:Protocols defined by this specification are:
    origin/master:spec.md:An implementation is not compliant for a given CPU architecture if it fails to satisfy one or more of the MUST, REQUIRED, or SHALL requirements for the protocols it implements.
    origin/master:spec.md:An implementation is compliant for a given CPU architecture if it satisfies all the MUST, REQUIRED, and SHALL requirements for the protocols it implements.

For example,

> An implementation is compliant for a given CPU architecture if it satisfies all the MUST, REQUIRED, and SHALL requirements for the platforms it implements.

reads a bit strangely to me, because we are currently including the CPU architecture in `platform`.  Maybe the protocol-less version of that line would be:


> An implementation is compliant for a given CPU architecture if it satisfies all the MUST, REQUIRED, and SHALL requirements for the operating systems it implements.

[1]: https://github.com/opencontainers/runtime-spec/pull/570#issue-177220681